src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -3171,6 +3171,11 @@ template<typename R, typename... Rx>
   INSN(fcvtas, 0, 0b00, 0b01, 0b11100);
   INSN(fcvtzs, 0, 0b10, 0b01, 0b11011);
   INSN(fcvtms, 0, 0b00, 0b01, 0b11011);
+  INSN(fcmgt,  0, 0b10, 0b01, 0b01100); // Floating-point compare greater than zero (vector)
```

Comment:
if you were to make this `fcm(Condition, ...` rather than having separate definitions for each condition it might make the code simpler and shorter.

Reply:
Thanks,  I think this would make the code much more simpler. But I was wondering if the function name in assembler_aarch64.hpp should align with ISA definition. 